wire-ios-data-model/Source/Model/UserClient/MockE2eIdentityProviders/MockExpiredE2eIdentityProvider.swift

```diff
@@ -25,7 +25,9 @@ public final class MockExpiredE2eIdentityProvider: E2eIdentityProviding {
 
     public var certificate: E2eIdentityCertificate {
         E2eIdentityCertificate(
-            certificateDetails: String(repeating: "abcdefghijklmno", count: 10),
+            certificateDetails: "BEGIN CERTIFICATE\n-----------\n"
+            + String(repeating: "abcdefghijklmno", count: 100)
+            +  "\n-----------\nEND CERTIFICATE",
```

Comment:
It might be interesting to move it to global constant since it's used in 3 other places